dashboard/config/environments/staging.rb

```diff
@@ -92,4 +92,8 @@
 
   # don't act like a levelbuilder by default
   config.levelbuilder_mode = CDO.with_default(false).levelbuilder_mode
+
+  # Whether or not to skip script preloading. Setting this to true
+  # significantly speeds up server startup time
+  config.skip_script_preload = false
```

Comment:
Given the relative number of times we hit staging vs. the number of times we rebuild staging, does it make sense for this to be true?